static/sass/_pattern_footer.scss

```diff
@@ -107,6 +107,16 @@
         font-size: .8125rem;
       }
 
+      span {
```

Comment:
Can you change this to use the `p-footer__title-text` class instead of a span, please? It would fit with the pattern better (and we try to avoid styling on element types).